requirements/base.txt

```diff
@@ -1,7 +1,7 @@
 # Requirements for app run
```

Comment:
@taniwha, you will also have to pin Django using `django~=2.2` because one of the dependencies that requires Django only specifies the minimum Django version.